src/common/Core/Hotkey.test.ts

```diff
@@ -30,12 +30,27 @@ describe(isValidHotkey, () => {
                 "Alt+]",
                 "Alt+(",
                 "Alt+)",
+                "Ctrl+Shift+Space",
+                "Alt+Shift+F2",
+                "Ctrl+Alt+Shift+Super+F8",
```

Comment:
Are you shure that `Ctrl+Alt+Shift+Super+F8` is working?

Reply:
See this article: https://www.electronjs.org/docs/latest/api/accelerator

Reply:
I tested it and it worked for me. I don't see anything from Electron docs that limit the number of modifiers keys?

Reply:
True, just tested it myself. Somehow I thought this doesn't work.